### ISSUE TITLE
[bug]: Tile ControlNet model for the chosen main model architecture [FLUX]

### ISSUE THREAD

**Issue by dannyvfilms** [invoke-ai/InvokeAI#7177]
### Is there an existing issue for this problem?

- [X] I have searched the existing issues

### Operating system

Windows

### GPU vendor

Nvidia (CUDA)

### GPU model

RTX 4060ti

### GPU VRAM

16GB

### Version number

5.3.0rc1

### Browser

Chrome

### What happened

![Screenshot 2024-10-23 194639](https://github.com/user-attachments/assets/d4f5f258-bbee-46a2-8cfe-2ffa80dbf340)
![Screenshot 2024-10-23 195122](https://github.com/user-attachments/assets/f3ce873d-ece5-4240-a08b-9bd52f8e19dd)

### What you expected to happen

After generating an image with FLUX, I right clicked the image and clicked Send to Upscale. Now that FLUX.1-dev-Controlnet-Union is available and installed, which explicitly says it includes Tile, I expected to be able to upscale an image using FLUX.

### How to reproduce the problem

Install the Starter Bundle of FLUX models, including FLUX.1-dev-Controlnet-Union, and attempt to use a FLUX model (Dev or Schnell) to upscale an image.

**Comment by maryhipp** [comment]
Upscaling tab is not supported for FLUX at this time

**Comment by vinipx** [comment]
@maryhipp I'm having the same issue for SDXL - Juggernaut XL V9

Should I open a different ticket? 

<img width="668" alt="Screenshot 2024-10-27 at 9 39 00 AM" src="https://github.com/user-attachments/assets/cc71a8ba-7a8f-4db8-89da-7537eae6f6d8">

**Comment by psychedelicious** [comment]
@vinipx Sorry for the late response. Yes, please create a separate issue.

**Comment by j-dbn** [comment]
For SDXL just install the "Tile" Controlnet Model for SDXL under "Starter Models". The message is then resolved once the model is done downloading.
